codersdk/users.go

```diff
@@ -92,6 +93,10 @@ type UpdateUserProfileRequest struct {
 	Username string `json:"username" validate:"required,username"`
 }
 
+type UpdateUserThemePreferenceRequest struct {
+	ThemePreference string `json:"theme_preference" validate:"required"`
```

Comment:
Should we validate a length, too? Or is it ok to set this to the empty string?

Reply:
empty string is a valid state. in the future we might allow administrators to set a custom default, so "user has never bothered to specify a theme" is distinct from "user explicitly chose a theme that happens to be the default"